packages/react-components/src/components/VideoTile.tsx

```diff
@@ -229,6 +261,27 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
   const canShowLabel = showLabel && (displayName || (showMuteIndicator && isMuted));
   const participantStateString = participantStateStringTrampoline(props, locale);
 
+  /* @conditional-compile-remove(pinned-participants) */
+  const optionsMenuItems = (menuItems: VideoTileMenuItems): IContextualMenuItem[] => {
+    const contextualMenuItems: IContextualMenuItem[] = [];
+    menuItems.forEach((item: VideoTileMenuItem) => {
+      contextualMenuItems.push({
+        key: item.key,
+        text: item.text,
+        onClick: item.onClick,
+        iconProps: item.icon
+      });
+    });
+
+    return contextualMenuItems;
+  };
+
+  const optionsIconStyles = mergeStyles({
+    minWidth: '2rem',
+    minHeight: '0rem'
+  });
+  const icon = (): JSX.Element => <MoreHorizontal20Filled key={'chatOnIconKey'} primaryFill="currentColor" />;
```

Comment:
this has a constant definition each render, pull this outside render body so its not redefined each render pass and not bloating the main render body

Reply:
In fact better would be to pull out the `<ControlBarButton` definition, e.g.:
```tsx
/** @private */
const VideoTileMoreOptionsButton = (props: {
  menuItems?: Array<{...}>
}) => {

  if (!menuItems || menuItems.length === 0) return <></>;

  const optionsMenuItems = (menuItems: VideoTileMenuItems): // etc...

  return (
    <ControlBarButton
      labelKey={'optionsButtonLabelKey'}
      onRenderOnIcon={icon}
      onRenderOffIcon={icon}
      menuIconProps={{ hidden: true }}
      menuProps={{ items: optionsMenuItems(menuItems) }}
      className={optionsIconStyles}
    />
}

const optionsIconStyles = mergeStyles({
  minWidth: '2rem',
  minHeight: '0rem'
});
```

Then in this render body you can do
```
/* @conditional-compile-remove(pinned-participants) */
<VideoTileMoreOptionsButton menuItems={props.menuItems} />
```